sdktests/helpers.go

```diff
@@ -42,13 +48,49 @@ func basicEvaluateFlag(
 	return result.Value
 }
 
+// computeExpectedBucketValue implements the bucketing hash value calculation as per the evaluation spec,
+// except that it returns the value as an integer in the range [0, 100000] - currently the SDKs convert
+// this to a floating-point fraction by in effect dividing it by 100000, but this test code needs an
+// integer in order to compute bucket weights.
+func computeExpectedBucketValue(
```

Comment:
This is identical to the v1 implementation in https://github.com/launchdarkly/sdk-test-harness/pull/56/files#diff-d49f6f508b72b350d5ce430e4f526a838bf90ced9c8a6070cea431494fb446ccR53.